README.md

```diff
@@ -1,41 +1,49 @@
 # cfn-square
-A CLI tool intended to simplify AWS CloudFormation handling. Derived from cfn-sphere.
 
 [![Build Status](https://travis-ci.org/KCOM-Enterprise/cfn-square.svg?branch=master)](https://travis-ci.org/KCOM-Enterprise/cfn-square)
 
-## cfn-sphere Features
+A CLI tool intended to simplify AWS CloudFormation handling. Originally forked from cfn-sphere.
+
+## Features
+
+### cfn-sphere
+
 - cfn templates in yml or json
 - build for human interaction and automation (run 'cf sync stacks.yml' triggered by a git push if you dare ;-)
-- a source of truth defining cloudformation stacks with their template and parameters
+- a source of truth defining CloudFormation stacks with their template and parameters
 - cross referencing parameters between stacks (use a stack output as parameter for another stack)
 - automatic stack dependency resolution including circular dependency detection
 - helper features easing the use of cfn functions like Fn::Join, Ref or Fn::GetAtt
 - easy user-data definition for https://github.com/zalando-stups/taupage
 - allow stack parameter values updates in command line interface 
 - encrypt/decrypt values with AWS KMS (https://aws.amazon.com/de/kms/)
 
-## cfn-square Additional Features
+### cfn-square additional features
 
-- Support for creating and executing cloudformation change sets
+- Support for creating and executing CloudFormation change sets
 
-## cfn-sphere Documentation
+## Install
 
-For original cfn-sphere features read:
+### As Python artifact:
 
-**https://github.com/cfn-sphere/cfn-sphere/wiki**
+    pip install cfn-square
 
-## cfn-square Documentation
+### Assumed role environment
 
-For cfn-square specific features read:
+cfn-square does not take a parameter to specify an AWS role to use. Therefore, before using cfn-square to modify AWS deployments, you must use a tool to bless the environment with a role. [roleshell](https://github.com/hashbrowncipher/aws-roleshell) and [awssume](https://github.com/manheim/awssume) are a couple of tools you can use to do this.
 
-**https://github.com/KCOM-Enterprise/cfn-square/wiki**
+#### roleshell
```

Comment:
Just realised this is a public repo - probably shouldn't link to our Sharepoint as the URLs can be useful inside info for attackers. Maybe remove the links but reference the title so it's easy to find?

Reply:
Thanks Nathan, fixed and pushed

Reply:
Ta - can you squash the commits to remove it from the branch's history?

Reply:
The commits will be squashed as part of the squash and merge when we merge this PR, won't they?

Reply:
Our current Git practice as it stands is to manually rebase to squash down to 1 commit before merging rather than relying on the "Squash and Merge" button.